doc/whats-new.rst

```diff
@@ -45,7 +45,11 @@ New Features
   By `Julia Signell <https://github.com/jsignell>`_.
 - :py:meth:`Dataset.where` and :py:meth:`DataArray.where` accept a lambda as a
   first argument, which is then called on the input; replicating pandas' behavior.
-  By `Maximilian Roos <https://github.com/max-sixty>`_
+  By `Maximilian Roos <https://github.com/max-sixty>`_.
+- implement ``skipna`` in :py:meth:`Dataset.quantile`, :py:meth:`DataArray.quantile`,
+  :py:meth:`Dataset.groupby.quantile`, :py:meth:`DataArray.groupby.quantile`
```

Comment:
```suggestion
  :py:meth:`core.groupby.DatasetGroupBy.quantile`, :py:meth:`core.groupby.DataArrayGroupBy.quantile`
```
otherwise the links are broken